src/components/errorNotification.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+
+interface ErrorNotificationProps {
+  error: string | null;
+}
+
+export const ErrorNotification: React.FC<ErrorNotificationProps> = ({
```

Comment:
file name with component should be in PascalCase - `ErrorNotification.tsx` in this case

Fix everywhere

Reply:
Not fixed